contrib/dist/linux/buildrpm.sh

```diff
@@ -95,48 +105,79 @@ orig_param="$@"
 #
 # usage information
 #
-usage="Usage: $0 [-b][-o][-m][-d][-u][-h] [-n <name>][-f <lf_location>][-t <tm_location>] <tarball>
+usage="Usage: $0 [-b][-o][-m][-d][-u][-s][-h] [-n name][-f lf_location][-t tm_location] tarball
+
+  -b
+             build all-in-one binary RPM only (required for all other flags to work)
+             {default: build only SRPM}
+
+  -n name  
+             name of the resulting RPM package set to "name". Requires "-b" flag.
+{default: openmpi}
+
+  -o         install in /opt/_NAME_/_VERSION_. Requires "-b" flag.
+             {default: install in /usr}
+
+  -m         install modulefiles during RPM installation. Requires "-b" flag.
+             {default: modulefiles will NOT be installed}
+
+  -i         build debuginfo RPM. Requires "-b" flag.
+             {default: do NOT build debuginfo RPM}
+
+  -f lf_location  
+             include Libfabric support from <lf_location>. Requires "-b" flag.
+             {default: build without Libfabric support}
```

Comment:
I think the default is to build Libfabric support only if it is found in default paths, right?  (i.e., it will *try* to build Libfabric support, but it won't error if Libfabric support is not found)